frontend/routes/components/textField/textField.styles.ts

```diff
@@ -15,14 +15,28 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import styled, { css } from 'styled-components';
+
 import { IconButton, InputLabel, TextField } from '@material-ui/core';
-import styled from 'styled-components';
+
+import { ContainedButton } from '../../viewerGui/components/containedButton/containedButton.component';
 import { LinkableField } from '../linkableField/linkableField.component';
 import { MarkdownField } from '../markdownField/markdownField.component';
 
+const containerEditModeStyles = css`
+	align-items: baseline;
+`;
+
+const containerNonEditModeStyles = css`
+	align-items: flex-end;
```

Comment:
do we need the flex-end at all? I played with this property on a few text fields and I'm not really seeing the significance.. Admittedly I'm also not too familiar with CSS 😆

Reply:
This is mainly about this case with the new copy button, and the way that buttons are disabled... because some times button are disabled by passing `disabled` property, and some times by using:
```js
inputProps={{
 readOnly: true
}}
```
One affects some style and display changes in general, the second one is just setting `readonly` attribute, which affects only in a functional way to the rendered input.

### disabled case
With `flex-end`
<img width="489" alt="Zrzut ekranu 2021-04-20 o 12 21 43" src="https://user-images.githubusercontent.com/24941866/115380600-06409800-a1d3-11eb-9816-b31797df1214.png">

With `baseline;`
<img width="480" alt="Zrzut ekranu 2021-04-20 o 12 22 51" src="https://user-images.githubusercontent.com/24941866/115384140-f925a800-a1d6-11eb-9484-bc1a8e08bc90.png">

### readonly case
With `flex-end`
<img width="969" alt="Zrzut ekranu 2021-04-20 o 12 52 29" src="https://user-images.githubusercontent.com/24941866/115384415-4dc92300-a1d7-11eb-97fe-fdb799f68f9b.png">

With `baseline`
<img width="960" alt="Zrzut ekranu 2021-04-20 o 12 51 40" src="https://user-images.githubusercontent.com/24941866/115384334-3722cc00-a1d7-11eb-895d-76034eeb47fd.png">